task_list-work-8526ea59.md

```
@@ -0,0 +1,29 @@
+# Task List — work-8526ea59
+
+## Implementation Tasks
+
+### Task 1: Apply fix to evaluate.rs:298
+- [ ] Replace line 298 in `crates/diffguard-domain/src/evaluate.rs` from:
+  ```rust
+  .and_then(|c| u32::try_from(c).ok());
+  ```
+  To:
+  ```rust
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)